src/components/VDataIterator/VDataIterator.spec.js

```diff
@@ -0,0 +1,74 @@
+import Vue from 'vue'
+import { test } from '~util/testing'
+import { mount } from 'avoriaz'
+import VDataIterator from './VDataIterator'
+
+test('VDataIterator.js', () => {
+  function dataIteratorTestData () {
+    return {
+      propsData: {
+        pagination: {
+          descending: false,
+          sortBy: 'col1',
+          rowsPerPage: 5,
+          page: 1
+        },
+        items: [
+          { other: 1, col1: 'foo', col2: 'a', col3: 1 },
+          { other: 2, col1: null, col2: 'b', col3: 2 },
+          { other: 3, col1: undefined, col2: 'c', col3: 3 }
+        ]
+      }
+    }
+  }
+
+  it('should match a snapshot - no matching records', () => {
```

Comment:
It's probably a good idea to check in these tests that the correct text is actually displayed.